src/Nokia.xml

```diff
@@ -0,0 +1,470 @@
+<spdx name="Nokia Open Source License" identifier="Nokia" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/nokia</url>
+  </urls>
+  <license>
+    <title>
+      <p>Nokia Open Source License (NOKOS License)</p>
+      <p>Version 1.0a</p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>DEFINITIONS.</p>
+          <p>&quot;Affiliates&quot; of a party shall mean an entity</p>
+        </li>
+        <list>
+          <li>
+            <b>a)</b>
+            <p>which is directly or indirectly controlling such party;</p>
+          </li>
+          <li>
+            <b>b)</b>
+            <p>which is under the same direct or indirect ownership or control as such party; or</p>
+          </li>
+          <li>
+            <b>c)</b>
+            <p>which is directly or indirectly owned or controlled by such party.</p>
+            <p>For these purposes, an entity shall be treated as being controlled by another if that other
+               entity has fifty percent (50%) or more of the votes in such entity, is able to direct its
+               affairs and/or to control the composition of its board of directors or equivalent
+               body.</p>
+            <p>&quot;Commercial Use&quot; shall mean distribution or otherwise making the Covered Software
```

Comment:
List indentation looks to have gone wrong here. Bullet for c) should end with the word 'party.' The text after this is just a new paragraph and not part of c) bullet.